src/clients/c++/experimental_api_v2/library/common.h

```diff
@@ -40,6 +40,9 @@
 #include <string>
 #include <thread>
 #include <vector>
+#ifdef _WIN32
+#include <algorithm>
```

Comment:
I this file requires something from <algorithm> we should just include it for all platforms